BE/src/main/java/com/sidedish/api/common/ResultDto.java

```diff
@@ -0,0 +1,23 @@
+package com.sidedish.api.common;
+
+import lombok.Getter;
+
+@Getter
+public class ResultDto {
+
+    private final Boolean valid;
+    private final String message;
+
+    public ResultDto(Boolean valid, String message) {
+        this.valid = valid;
+        this.message = message;
+    }
+
+    public static ResultDto ok() {
+        return new ResultDto(true, "ok");
```

Comment:
하드코딩보다는 HttpStatus를 사용하시는게 좋을 듯 하네요.
```suggestion
        return new ResultDto(true, HttpStatus.OK.name());
```